framework/doc/content/source/meshgenerators/ConcentricCircleMeshGenerator.md

```diff
@@ -4,24 +4,41 @@
 
 ## Description
 
-The `ConcentricCircleMeshGenerator` object is to generate meshes for concentric circles. Particularly, it would be useful to generate 2D meshes for typical LWR fuels. Users can refine meshes by providing a higher number of rings for each specified concentric circle region or a higher number of sectors in each quadrant.
+The `ConcentricCircleMeshGenerator` object is to generate mesh for concentric circles. Specifically, it would be useful to generate 2D mesh. Users can refine mesh by providing a high number of rings for each specified concentric circle region or a high number of sectors in each quadrant. The test, [concentric_circle_mesh_generator.i](test/tests/meshgenerators/concentric_circle_mesh_generator/concentric_circle_mesh_generator.i), explains the use of the 'ConcentricCircleMeshGenerator' object. To execute the test only for generating 2D mesh,
```

Comment:
> Specifically, it would be useful to generate 2D meshes

It only generates 2D meshes. I'd remove the sentence

Reply:
This has been removed.